app/src/main/java/com/example/plantsapp/domain/repository/PlantsRepository.kt

```diff
@@ -1,11 +1,14 @@
 package com.example.plantsapp.domain.repository
 
 import com.example.plantsapp.domain.model.Plant
+import com.example.plantsapp.domain.model.Task
 import kotlinx.coroutines.flow.Flow
+import java.util.Date
 
 interface PlantsRepository {
 
     suspend fun fetchPlants(): Flow<List<Plant>>
     suspend fun addPlant(plant: Plant)
     suspend fun getPlantByName(name: Plant.Name): Plant
+    suspend fun getTasksForDate(date: Date): Flow<List<Task>>
```

Comment:
Lets create separate Task repository - maybe it is not needed as much, but our Plants can leave without tasks, so for example repository may not know about Tasks at all